py/torch_tensorrt/fx/fx2trt.py

```diff
@@ -251,6 +266,34 @@ def run(
         engine = self.builder.build_engine(self.network, builder_config)
         assert engine
 
+        import os
+        def get_file_name(org):
+            file_name = org
+            i = 0
+            while os.path.exists(os.path.abspath(file_name)):
+                i += 1
+                file_name = org + str(i)
+            return file_name
+
+        engine_file = os.environ.get('TORCH_FX_DUMP_ENGINE')
```

Comment:
We will need to check with Torch-TRT team to see if they are okay with this approach to control engine saving using envvar, or would they prefer to convert this into another LowerSetting.

Reply:
I think this would be better outside of the compile pipeline. I could imagine a piece of code which takes in a compiled module and recursively finds all TRTModule / TRTModuleNext and extracts the engines.

Reply:
I am happy to include this as a utility function in the library. Something like `torch_tensorrt.fx.extract_engines_from_module(mod: torch.fx.GraphModule) -> List[bytearray]` or something

Reply:
okay, @wu6u3tw let's remove the engine dump and layer info dump code from this PR for now. We can do that in another PR. Thanks

Reply:
Done.